tests/__mocks__/results/permit-generation-results.json

```diff
@@ -2551,6 +2695,40 @@
           "reward": 25
         }
       }
-    ]
+    ],
+    "events": {
+      "issue.commented": {
+        "count": 1,
+        "reward": 0
+      },
+      "issue.mentioned": {
+        "count": 3,
+        "reward": 0
+      },
+      "issue.subscribed": {
+        "count": 3,
+        "reward": 0
+      },
+      "pull_request.review_comment": {
+        "count": 11,
+        "reward": 0
+      },
+      "pull_request.reviewed.approved": {
+        "count": 1,
+        "reward": 0
+      },
+      "pull_request.reviewed.changes_requested": {
+        "count": 1,
+        "reward": 0
+      },
+      "reaction.received.+1": {
+        "count": 3,
+        "reward": 0
+      },
+      "reaction.sent.+1": {
```

Comment:
After reading the ask response, yeah, I think sent and received is the cleanest for sure. Good work!